lld/ELF/Thunks.h

```diff
@@ -9,8 +9,8 @@
 #ifndef LLD_ELF_THUNKS_H
 #define LLD_ELF_THUNKS_H
 
-#include "llvm/ADT/SmallVector.h"
 #include "Relocations.h"
+#include "llvm/ADT/SmallVector.h"
```

Comment:
just to bring attention to include-sorting! This can technically change semantics for poorly written include files, but I personally like the setting (and also the incentive to not have poorly written include files)...